task.sql

```diff
@@ -1 +1,38 @@
-# Write your SQL code for the database creation here. Good luck! 
+CREATE DATABASE ShopDB;
+
+USE ShopDB;
+
+CREATE TABLE Products (
+    ProductID INT AUTO_INCREMENT,
+    Name VARCHAR(50),
+    Description VARCHAR(50),
+    Price VARCHAR(50),
+    WarehouseAmount VARCHAR(50),
```

Comment:
The 'Price' column is defined as VARCHAR(50). Typically, prices should be stored as a numeric type, such as DECIMAL, to allow for proper arithmetic operations and to ensure data integrity.